netket/stats/mc_stats.py

```diff
@@ -51,6 +46,13 @@ def _format_decimal(value, std, var):
 _NaN = float("NaN")
 
 
+def _maybe_item(x):
+    if hasattr(x, "shape") and x.shape == ():
+        return x.item()
+    else:
+        return x
```

Comment:
Can we fix this in the constructor rather than make it a mess to use it down the road?

Reply:
I don't think so: Calling `item()` to get a scalar value without wrapping `DeviceArray` does not work under JIT, but `Stats` needs to be constructed within a JITed function.